src/wui/mapview.cc

```diff
@@ -324,6 +324,8 @@ MapView::MapView(
      dragging_(false),
      edge_scrolling_(parent && !parent->get_parent() /* not in watch windows */ &&
                      get_config_bool("edge_scrolling", false)),
+     invert_movement_(parent && !parent->get_parent() /* not in watch windows */ &&
+                     get_config_bool("invert_movement", false)),
```

Comment:
This feature should be used for watch windows as well for consistency:
```suggestion
     invert_movement_(get_config_bool("invert_movement", false)),
```